### ISSUE TITLE
EventEmitter memory leak detected

### ISSUE THREAD

**Issue by terrybleger** [node-inspector/node-inspector#543]
Hi,

So far I've got no problem with debugging. But this error is always shown in chrome and terminal.

It's clearly not from my code: **```V8Debug.addListener```**

```
(node) warning: possible EventEmitter memory leak detected. 11 close listeners added. Use emitter.setMaxListeners() to increase limit.
Trace
    at Console.eval (eval at WRAPPED_BY_NODE_INSPECTOR (/usr/local/lib/node_modules/node-inspector/node_modules/v8-debug/v8-debug.js:113:15), <anonymous>:123:19)
    at Console.eval [as _trace] (eval at WRAPPED_BY_NODE_INSPECTOR (/usr/local/lib/node_modules/node-inspector/node_modules/v8-debug/v8-debug.js:113:15), <anonymous>:113:21)
    at Console.eval [as trace] (eval at WRAPPED_BY_NODE_INSPECTOR (/usr/local/lib/node_modules/node-inspector/node_modules/v8-debug/v8-debug.js:113:15), <anonymous>:123:19)
    at V8Debug.addListener (events.js:179:15)
    at v8-profiler (eval at WRAPPED_BY_NODE_INSPECTOR (/usr/local/lib/node_modules/node-inspector/node_modules/v8-debug/v8-debug.js:113:15), <anonymous>:57:9)
    at eval (eval at WRAPPED_BY_NODE_INSPECTOR (/usr/local/lib/node_modules/node-inspector/node_modules/v8-debug/v8-debug.js:113:15), <anonymous>:63:3)
    at DebugCommandProcessor.WRAPPED_BY_NODE_INSPECTOR [as processDebugRequest] (/usr/local/lib/node_modules/node-inspector/node_modules/v8-debug/v8-debug.js:113:15)
```

node v0.11.16, npm v2.4.1, osx 10.10.2

**Comment by 3y3** [comment]
This happens after first debugger attachment?

**Comment by terrybleger** [comment]
Since the only way that breakpoint can work is I have to refresh the browser that point to debugger url (127.0.0.1:8080) after I added the breakpoint or every time I added new breakpoint.

I have a code that uses ```EventEmitter```, and that code already listen to more than the default node's maxListeners, in my code I've already call the ```setMaxListeners()```, so I won't get any crash/error.

The error message will show after I refresh the browser the second time, the first time is to make the breakpoint work, the second time (after the code being executed), because every time I add new breakpoint I have to refresh it. And the error only shown once. The only way I can reproduce it is by restarting my server, and doing the thing I said before.

But I've no problem on debugging my code anyway.

**Comment by 3y3** [comment]
> The way I reproduce it is just by refreshing the browser

Thanks, now I know where to search.

**Comment by 3y3** [comment]
Way to fix this:
1. Deprecate multi connection system in node 0.12+ debugger.
2. Emulate disconnect message on client connection closing.

Also can be helpful to send real disconnect command from node-inspector.
But anyway regression in nodejs needs to be fixed.

**Comment by 3y3** [comment]
Fixed in 0.11.0

**Comment by amurchick** [comment]
Still not fixed (
```
$ node -v
v5.10.1

$ node-inspector -v
Node Inspector v0.12.7
```
```
error: [ConsoleAgent.js:95] (node) warning: possible EventEmitter memory leak detected. 11 connect listeners added. Use emitter.setMaxListeners() to increase limit.
error: [ConsoleAgent.js:95] Trace
    at Console.WRAPPED_BY_NODE_INSPECTOR [as trace] (/usr/local/lib/node_modules/node-inspector/lib/Injections/ConsoleAgent.js:95:19)
    at Socket.addListener (events.js:252:17)
    at Socket.Readable.on (_stream_readable.js:673:33)
    at Socket.once (events.js:278:8)
    at ClientRequest.handleSocket (/usr/local/lib/node_modules/node-inspector/lib/Injections/NetworkAgent.js:267:12)
    at ClientRequest.g (events.js:273:16)
    at emitOne (events.js:95:20)
    at ClientRequest.emit (events.js:182:7)
    at tickOnSocket (_http_client.js:541:7)
    at onSocketNT (_http_client.js:553:5)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```
